internal/pkg/levee/testdata/src/example.com/tests/binop/tests.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package binop
+
+import (
+	"fmt"
+
+	"example.com/core"
+)
+
+func TestConcatenatingTaintedAndNonTainteStrings(prefix string) {
+	s := core.Source{Data: "password1234"}
+	message := fmt.Sprintf("source: %v", s)
+	fullMessage := prefix + message
+	// TODO: no report should be produced for "prefix"
+	core.Sink(prefix)      // want "a source has reached a sink"
```

Comment:
I thought we were adding the TODO here?

Reply:
Good callout. For `want` tests that is true, I agree that `TODO want` should be preferred, and indeed that is the way all such `TODO`'s are written (to my knowledge). This case however is what I would call a `do not want` test, and for those I usually put the `TODO` on the line above (see e.g. `testdata/src/example.com/tests/sanitization/tests.go`).  I could put it in the same comment as the `want`, but I'm worried it would be less readable that way.

WDYT?

Reply:
That seems reasonable to me. Thanks!